x-pack/plugins/security_solution/public/timelines/components/timeline/body/actions/index.tsx

```diff
@@ -217,11 +217,12 @@ const ActionsComponent: React.FC<ActionProps> = ({
   );
 
   const onExpandEvent = useCallback(() => {
-    if (isTourAnchor) {
+    const isStep2Active = activeStep === 2 && isTourShown(SecurityStepId.alertsCases);
```

Comment:
NIT: It would be nice to have an enum or some constant to make these step numbers more comprehensible, something like:
`const isReviewAlertsTourStepActive = activeStep === AlertsCasesTourSteps.reviewAlerts && isTourShown(SecurityStepId.alertsCases);` (not sure if step 2 is "review alerts" though)

It would also make it much easier to do changes or add steps in the future, because we wouldn't have to go and update the numbers in all the components, only the numbers in the enum constant.

Reply:
yeah i thought about going that direction, but I had trouble naming the steps. It was easier to me to think of them as step 1, step 2, step 3 than `AlertsCasesTourSteps.pointToAlertName`, `AlertsCasesTourSteps.expandEvent`, `AlertsCasesTourSteps.reviewAlertDetailsFlyout`. Also this made it easy on the `incrementStep` function to know which step is next

Reply:
that makes a lot of sense, ill make that change. thanks for explaining!